### PR TITLE
Fix the dependency relationship for the perspective plugin

### PR DESCRIPTION
**Pull request by wpietri**
…ets updated.

### CODE REVIEW

**Review by brianwgoldman — APPROVED** [review]

**Review by dhosterman — APPROVED** [review]
That should do it!

File: plugins/standard_tests/pyproject.toml [hunk 1]
```diff
@@ -8,8 +8,7 @@ packages = [{include = "newhelm"}]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-# TODO: This probably shouldn't be here.
-perspective_api = {version = "0.1.1", optional = true}
+perspective_api = {version = "*"}
```
Comment (brianwgoldman) [inline plugins/standard_tests/pyproject.toml:11]: Consider also running `poetry lock` to update the lock file.

Comment (wpietri) [reply plugins/standard_tests/pyproject.toml:11]: Oh, duh, right.

Comment (wpietri) [reply plugins/standard_tests/pyproject.toml:11]: Oops! Great point.

**Pull request merged**